modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
+The sum of outbound requests in the period, grouped by endpoint.
+
+Outbound - Response Time (99, 90, 75, and 50 percentile)::
+The average response time of the 99, 90, 75, and 50 percentiles of outbound requests, grouped by application in automatic intervals. The unit for these metrics is milliseconds (ms).
+
+Outbound - Failure Total Request Count::
+The sum of failed outbound requests, grouped by type of response in automatic intervals.
+
+Outbound - Ports::
+The sum of outbound remote port, grouped by endpoint in automatic intervals.
+
+// ports?
+
+Outbound - Operations::
+The sum of outbound requests, grouped by operation in automatic intervals.
+
+Outbound - Protocols::
+The sum of outbound requests, grouped by protocol in automatic intervals.
+
+Outbound - Endpoints::
+The sum of outbound requests, grouped by remote hosts (endpoints) in automatic intervals.
+
+[[flows-charts]]
+== Flows
+
+The following charts appear on the *Flows* dashboard:
+
+Inbound Response Time by Flow::
+The average inbound response time, grouped by flow.
+
+Outbound Response Time by Flow::
+The average outbound response time, grouped by flow.
+
+Inbound Request Volume by Flow::
+The average number of inbound requests, grouped by flow.
+
+Outbound Request Volume by Flow::
+The average number of outbound requests, grouped by flow.
+
+Failures by Flow::
+The average number of failed inbound requests, grouped by flow.
+//tbd?
+
+Slow Inbound Requests by Flow::
+The average number of inbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Slow Outbound Requests by Flow::
+The average number of outbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Inbound Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
+
+Outbound Response Time by Flow::
+The average, minimum, and maximum outbound response time, grouped by flow.
+
+Inbound Requests by Flow::
+The average number of inbound requests, grouped by flow.
+
+Outbound Requests by Flow::
+The average number of outbound requests, grouped by flow.
+
+Failures by Flow::
+The average number of inbound failed requests, grouped by flow.
+//tbd?
+
+Slow Inbound Requests by Flow::
+The average number of inbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Slow Outbound Requests by Flow::
+The average number of outbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+[[connectors-charts]]
+== Connectors
+
+If you don't see the *Connectors* tab, verify that you have a Titanium subscription to Anypoint Platform and that the app you selected is a hybrid app deployed to a server, server group, or cluster. See xref:monitor-connectors.adoc[Monitor Connectors].
+
+The following charts appear on the *Connectors* dashboard:
+
+Connector Message Volume::
+The sum of outbound messages and outbound requests grouped by connector type in automatic intervals.
```

Comment:
The sum of outbound requests grouped by connector type in automatic intervals.

Reply:
```suggestion
The sum of outbound requests grouped by connector type in automatic intervals.
```